### PR TITLE
Bepro 1636 mobile curators page inside a network

### PR DESCRIPTION
**Pull request by vhcsilva**

### CODE REVIEW

File: components/curators-list.tsx [hunk 1]
```diff
@@ -1,127 +1,67 @@
...
-    if (page) {
-      const pagesToValidate = [...Array(+page).keys()].map((i) => i + 1);
-
-      setTruncatedData(!pagesToValidate.every((pageV) =>
-          curatorsPage.find((el) => el.page === pageV)));
-    }
-  }, [page, curatorsPage]);
-
-  if(inView !== null && inView === false) return null;
+    if (!curators) return;
```
Comment (moshmage) [inline components/curators-list.tsx:31]: wouldn't it make sense to have this list come into the component _already parsed_?

File: components/lists/sort/controller.tsx [hunk 2]
```diff
@@ -29,10 +24,10 @@ export default function ListSort({
       page: "1"
     };
 
-    router.push({ pathname: router.pathname, query }, router.asPath, { shallow: false, scroll: false });
+    router.push({ pathname: `${router.pathname}`, query }, router.asPath, { shallow: false, scroll: false });
```
Comment (moshmage) [inline components/lists/sort/controller.tsx:27]: we can use `router.pathname.toString()` instead, we don't need to template-string it

File: components/lists/sort/controller.tsx [hunk 3]
```diff
@@ -7,24 +9,32 @@ import { CustomDropdownItem, SortOption } from "types/components";
...
+
     const query = {
       ...router.query,
       sortBy: newValue.sortBy,
       order: newValue.order,
       page: "1"
     };
 
-    router.push({ pathname: `${router.pathname}`, query }, router.asPath, { shallow: false, scroll: false });
+    router.push({ pathname: router.pathname, query }, router.asPath, { shallow: false, scroll: false });
```
Comment (moshmage) [inline components/lists/sort/controller.tsx:37]: if we route.push with `shallow: false` the page will reload, thus the `setSelectedIndex` will be replaced with the one from the refreshed page

**Review by moshmage — CHANGES_REQUESTED** [review]

File: components/lists/sort/controller.tsx (hunk 3 above)
Comment (vhcsilva) [reply components/lists/sort/controller.tsx:37]: @moshmage the page won't be reloaded, the `shallow` property only prevents the `getServerSideProps` to be executed, with `shallow: false` I'm ensuring that will run with the new `sortBy` and `order` values

**Review by moshmage — DISMISSED** [review]

**Review by moshmage — APPROVED** [review]

**Pull request merged**